.github/CODEOWNERS

```diff
@@ -308,11 +308,12 @@
 /pkg/config/env                         @DataDog/container-integrations @DataDog/container-app
 /pkg/config/logs                        @Datadog/agent-shared-components @Datadog/agent-platform
 /pkg/config/logs/internal/seelog/seelog_config.go          @Datadog/agent-shared-components
-/pkg/config/process*.go                 @DataDog/processes
-/pkg/config/system_probe.go               @DataDog/ebpf-platform
-/pkg/config/system_probe_cws.go           @DataDog/agent-security
-/pkg/config/system_probe_cws_notwin.go    @DataDog/agent-security
-/pkg/config/system_probe_cws_windows.go   @DataDog/windows-kernel-integrations
+/pkg/config/setup                            @DataDog/agent-shared-components
+/pkg/config/setup/process*.go                 @DataDog/processes
+/pkg/config/setup/system_probe.go               @DataDog/ebpf-platform
+/pkg/config/setup/system_probe_cws.go           @DataDog/agent-security
+/pkg/config/setup/system_probe_cws_notwin.go    @DataDog/agent-security
+/pkg/config/setup/system_probe_cws_windows.go   @DataDog/windows-kernel-integrations
```

Comment:
Please update `/pkg/config/apm.go` as well (it's been renamed in this PR).

Reply:
Done